packages/zowe-explorer/i18n/sample/src/utils/TempFolder.i18n.json

```diff
@@ -1,4 +1,5 @@
 {
   "moveTempFolder.error": "Error encountered when creating temporary folder!",
-  "deactivate.error": "Unable to delete temporary folder. "
+  "deactivate.error": "Unable to delete temporary folder. ",
+  "findRecoveredFiles.message": "One or more files remained open in your last VS Code session. To prevent losing your updates, check the Problems view to see the list of files and re-save them to upload edits to the mainframe."
```

Comment:
Suggesting two small edits: "To **avoid** losing your updates ... re-save them to upload **the changes** to the mainframe."